webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -226,7 +226,6 @@ public function viewAction(Request $request, string $execId)
 
     /**
      * @Route("/{execId}/content", name="jury_executable_content")
-     * @IsGranted("ROLE_ADMIN")
```

Comment:
Don't we need to keep ROLE_JURY here (and in the other places)?

Reply:
It has this on the whole controller level already

Reply:
Oh, I see now that this is on top of this file.